validation/standard-config.toml

```
@@ -0,0 +1,9 @@
+# Standard Chain Config Parameters
+[resource_config]
+
+max_resource_limit = 20000000
+elasticity_multiplier = 10
+base_fee_max_change_denominator = 8
+minimum_base_fee = 1000000000
+system_tx_max_gas = 1000000
+maximum_base_fee = "340_282_366_920_938_463_463_374_607_431_768_211_455"
```

not a blocker, but can we support more human readable values? (guessing we have to use a string for the last one)
```suggestion
max_resource_limit = 2e7
elasticity_multiplier = 10
base_fee_max_change_denominator = 8
minimum_base_fee = 1e9
system_tx_max_gas = 1e6
maximum_base_fee = "340_282_366_920_938_463_463_374_607_431_768_211_455" # 2^128 - 1
```

---

This is possible, but some amount of work since it won't work out of the box. We can circle back to this when we tackle things like ranges  `[a,b]` and wildcards `*`.

---

I've added a link to this idea in  #168